packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -107,4 +108,22 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection {
 			url: getUrlForTelemetry(this.url, socketIoPath),
 		};
 	}
+
+	/**
+	 * Disconnect from the websocket
+	 */
+	protected disconnectCore(err: IAnyDriverError): void {
+		const isCorruption =
+			err.errorType === FluidErrorTypes.dataCorruptionError ||
+			err.errorType === FluidErrorTypes.dataProcessingError;
```

Comment:
I don't think a data processing error should result in `isCorruption = true`. The client might not be able to process an op for some reason, but the problem could be limited to that client (e.g. it's still on an older version than others, and it doesn't understand an op it received). Not the expert in this area but we should check with the rest of the team to make sure what we say in the event is correct.

Reply:
We have decided to just pass along the errorType as a string to frs, and frs can compuse what they consider abnormal